test/e2e/helpers.js

```diff
@@ -50,6 +54,40 @@ async function selectOption(selector, textOrIndex = 'random', cssSelector) {
   return option
 }
 
+export function generatePerson({ pncNumber } = {}) {
+  return {
+    police_national_computer:
+      pncNumber ||
+      faker
+        .fake('{{random.alphaNumeric(6)}}/{{random.alphaNumeric(2)}}')
+        .toUpperCase(),
+    prison_number: faker.fake(
+      '{{helpers.replaceSymbols("?")}}{{random.number}}{{helpers.replaceSymbols("??")}}'
+    ),
+    last_name: faker.name.lastName(),
+    first_names: faker.name.firstName(),
+    date_of_birth: format(
+      faker.date.between('01-01-1940', '01-01-1990'),
+      'yyyy-MM-dd'
+    ),
+  }
+}
+
+export async function createPersonFixture() {
+  const person = await personService.create(generatePerson())
+
+  return {
+    ...person,
+    fullname: `${person.last_name}, ${person.first_names}`.toUpperCase(),
+    prison_number: find(person.identifiers, {
+      identifier_type: 'prison_number',
+    }).value,
+    police_national_computer: find(person.identifiers, {
+      identifier_type: 'police_national_computer',
+    }).value,
+  }
+}
+
 /**
  * Select random option from autocomplete menu
  *
```

Comment:
Can't see how to apply the comment to the correct line (101), but suggest changing

`await t.click(fieldSelector)`

to

```
  const autocompleteClicker = fieldSelector.sibling('div').find('input')
  await t.click(autocompleteClicker)
```

which seems robust and does not depend on other events on the page preceding it